cmd/registry/patterns/resources.go

```diff
@@ -406,34 +411,34 @@ func (ar ArtifactResource) ResourceName() ResourceName {
 	return ar.ArtifactName
 }
 
-func ListResources(ctx context.Context, client connection.RegistryClient, pattern, filter string) ([]ResourceInstance, error) {
+func ListResources(ctx context.Context, client connection.RegistryClient, pattern, filter string, preserveResource bool) ([]ResourceInstance, error) {
```

Comment:
Done.

Right, I had initially added this flag because I thought that the controller is also making use of the `ListResources ` function, to avoid refactoring. But the controller has it's own implementation of the lister interface and I missed taking the flag out.